Integrations/integration-Carbon_Black_Enterprise_Response.yml

```diff
@@ -552,12 +555,15 @@ script:
 
         title = 'Binaries';
         headers = ['MD5', 'Publisher', 'Result'];
-        let keys = Object.keys(result.process.binaries);
-        translatedData = keys.map(md5 => ({
-            'MD5': md5,
-            'Publisher': result.process.binaries[md5]['digsig_publisher'],
-            'Result': result.process.binaries[md5]['digsig_result']
-        }));
+        translatedData = [];
+        if (result.process.binaries) {
```

Comment:
Please make sure we have `process` in `result` or this line would fail

Reply:
done